wd_helper.au3

```diff
@@ -509,8 +509,8 @@ EndFunc   ;==>_WD_IsWindowTop
 ;                  @ERROR       - $_WD_ERROR_Success
 ;                  				- $_WD_ERROR_Exception
 ; Author ........: Decibel
-; Modified ......: 2018-04-27
-; Remarks .......:
+; Modified ......: mLipok
+; Remarks .......: For nested frames use this function as many time as many nesting level is in analyzed website

```

Comment:
`You can drill-down into nested frames by calling this function repeatedly with the correct parameters`
will be better

Reply:
@mlipok Pretty sure that you just submit an additional commit to the same branch (https://github.com/mlipok/WebDriver/tree/patch-4)

Reply:
I hope I do exactly what you mean, if not please provide me a hint.